website/pages/blog/announcing-cloudquery-aws-event-based-sync-beta.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Announcing Closed Beta of CloudQuery AWS Plugin with Event-based Sync
+tag: announcement
+date: 2023/08/31
+description:
+author: michal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+We are excited to announce the preview of an event-based sync for our [AWS Plugin](https://www.cloudquery.io/docs/plugins/sources/aws/overview)!
+
+## What is it?
+
+The way CloudQuery has been initially designed was to sync all data on demand to give you the full control over what to sync, when, and how often. This approach makes CloudQuery easy to set up and use.
+
+However, regular sync, even if it runs every hour, is sometimes just not enough to get the accurate picture of what is happening in your environment. Cloud environments are ephemeral - they come and go in just a few minutes and it is really hard to track them and get your costs right. An IP address gets spammed by bots trying to find their way in the moment you make it public. User accounts get created with too many permissions and can get misused in a brief moment.
+
+This is where our new event-based sync comes to rescue.
+
+<div style={{marginTop:"2em", align: "center"}}><a href="https://cloudquery.typeform.com/to/Dvdn6P2u" class="btn btn-blue">Sign up for early access</a></div>
+
+## How it works
+
+All events are aggregated by AWS CloudTrail. You can configure a Trail to send management events to a Kinesis Data Stream via CloudWatch Logs. By subscribing to a stream of AWS CloudTrail events in the Kinesis Data stream, CloudQuery can then trigger selective syncs to update just the singular resource that had a configuration change.
+
+![Configuring CloudQuery AWS Plugin for event-based sync](/images/blog/announcing-aws-event-based-sync/diagram.png)
```

Comment:
Nit: I'm wondering if the diagram should include the databases? Ie. not stop at CloudQuery, and show some of the main databases CQ would sync to

Reply:
good idea, updated.